packages/components/package.json

```diff
@@ -15,8 +15,8 @@
     "test:unit": "jest --coverage",
     "lint": "run-p lint:*",
     "lint:ts": "eslint --fix src/**/*.ts*",
-    "lint:styles": "stylelint src/**/*.scss",
-    "build-storybook": "build-storybook -s ./static -c .storybook -o .out"
+    "lint:styles": "stylelint --fix src/**/*.scss",
+    "build-storybook": "build-storybook -s ./static -c .storybook -o ../client/build/storybook && pwd && ls ../client/build/storybook"
```

Comment:
TODO: we still need to remove the pwd & ls from the above line

Reply:
Ah, I didn't realise it was in there as well. I'll take it out now.